test/node-unit/cli/mocha-flags.spec.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+const {
+  types,
+  expectedTypeForFlag
+} = require('../../../lib/cli/run-option-metadata');
+
+describe('mocha-flags', function () {
+  describe('expectedTypeForFlag()', function () {
+    it('returns expected type for all mocha flags', function () {
+      Object.entries(types).forEach(([dataType, flags]) => {
+        flags.forEach(flag => {
```

Comment:
[Tests] The problem with doing >=1 `.forEach` / loop in general _inside_ a test is that you only get one complaint if _any_ of the cases fail. But it's generally nicer to have a test for each of the cases. That way you get a nice detailed log of all the cases and which ones fail.

We can get that with a cute lil reorder and templatization of the `it` name:

```suggestion
    Object.entries(types).forEach(([dataType, flags]) => {
      flags.forEach(flag => {
        it(`returns expected ${flag}'s type as ${dataType}`, function () {
```